### PR TITLE
Make sure the correct default presentation attribute is returned

### PR DESCRIPTION
**Pull request by mvanbeusekom**
This change ensures the `GetPresentationAttributes` method returns the correct presentation attribute in case there is no presentation attribute found on the ViewController. In case there is no `MvxNavigationController` instance is available it will return a `MvxRootPresentationAttribute` otherwise a `MvxChildPresentationAttribute` is returned.

This will solve the problem where the `MvxException` (with the message "Trying to show View type: CenterPanelView as child, but there is no current Root!") is being thrown.

### CODE REVIEW

File: MvvmCross/iOS/iOS/Views/Presenters/Attributes/MvxRootPresentationAttribute.cs [hunk 1]
```diff
@@ -3,6 +3,6 @@ namespace MvvmCross.iOS.Views.Presenters.Attributes
 {
     public class MvxRootPresentationAttribute : MvxBasePresentationAttribute
     {
-        public bool WrapInNavigationController { get; set; }
+		public bool WrapInNavigationController { get; set; } = true;
```
Comment (nmilcoff) [inline MvvmCross/iOS/iOS/Views/Presenters/Attributes/MvxRootPresentationAttribute.cs]: I'm not sure about making this member `true` by default, Because by doing so, when a user tries to set a `TabBarController` or a `SplitViewController` as root, he will have to set this to `false` explicitly

File: MvvmCross/iOS/iOS/Views/Presenters/MvxIosViewPresenter.cs [hunk 2]
```diff
@@ -371,12 +371,19 @@ protected virtual void SetWindowRootViewController(UIViewController controller)
...
             {
-                MvxTrace.Trace($"PresentationAttribute not found for {viewController.GetType().Name}, assuming animated Child presentation");
-                attributes = new MvxChildPresentationAttribute();
+				return attributes;
             }
-            return attributes;
+
+			if (MasterNavigationController == null)
+			{ 
+				MvxTrace.Trace($"PresentationAttribute nor MasterNavigationController found for {viewController.GetType().Name}. Assuming animated Root presentation");
```
Comment (nmilcoff) [inline MvvmCross/iOS/iOS/Views/Presenters/MvxIosViewPresenter.cs]: Root changes are not being animated currently 😬

File: MvvmCross/iOS/iOS/Views/Presenters/Attributes/MvxRootPresentationAttribute.cs (hunk 1 above)
Comment (mvanbeusekom) [inline MvvmCross/iOS/iOS/Views/Presenters/Attributes/MvxRootPresentationAttribute.cs]: I see that would not be nice indeed. I have added a new commit correcting this problem.

Comment (nmilcoff) [inline MvvmCross/iOS/iOS/Views/Presenters/Attributes/MvxRootPresentationAttribute.cs]: Mhm but... the commit does not revert this change. `MvxRootPresentationAttribute` is the one that needs to be modified

Comment (mvanbeusekom) [inline MvvmCross/iOS/iOS/Views/Presenters/Attributes/MvxRootPresentationAttribute.cs]: Sorry committed the wrong file. Will fix iT now

Comment (mvanbeusekom) [inline MvvmCross/iOS/iOS/Views/Presenters/Attributes/MvxRootPresentationAttribute.cs]: Fixed, that was a stupid mistake.

**Comment by mvanbeusekom** [comment]
Fixed the spacing issues, tried to squash changes but not sure it worked out correctly.

**Comment by kjeremy** [comment]
Squash all of the commits into one (use `git rebase -i` to help) and then do a `git push --force`

**Comment by mvanbeusekom** [comment]
commits are squashed
